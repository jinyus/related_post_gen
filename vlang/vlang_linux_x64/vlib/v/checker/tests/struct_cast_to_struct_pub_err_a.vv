struct Abc {
pub:
	name string
}

struct Xyz {
	name string
}

fn main() {
	abc := Abc{}
	_ := Xyz(abc)
}
