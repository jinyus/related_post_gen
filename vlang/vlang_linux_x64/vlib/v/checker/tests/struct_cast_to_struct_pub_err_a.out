vlib/v/checker/tests/struct_cast_to_struct_pub_err_a.vv:12:7: warning: casting to struct is deprecated, use e.g. `Struct{...expr}` instead
   10 | fn main() {
   11 |     abc := Abc{}
   12 |     _ := Xyz(abc)
      |          ~~~~~~~~
   13 | }
vlib/v/checker/tests/struct_cast_to_struct_pub_err_a.vv:12:7: error: cannot convert struct `Abc` to struct `Xyz`
   10 | fn main() {
   11 |     abc := Abc{}
   12 |     _ := Xyz(abc)
      |          ~~~~~~~~
   13 | }
